client/app/states/services/details/details.html

```diff
@@ -199,7 +199,7 @@ <h2 translate>Virtual Machines ({{ ::vm.vms.length }})</h2>
                   <i class="fa fa-html5 fa-lg"></i>
                 </button>
                 <button class="btn btn-default open-console-button" ng-if="item['supports_cockpit?'] && item.power_state == 'on'" tooltip="{{'Open Cockpit console for this VM'|translate}}" tooltip-placement="left">
-                  <i class="fa fa-html5 fa-lg"></i>
+                  <img src="/client/assets/images/cockpit.png"></img>
```

Comment:
This is the same icon that we have in the operations UI and this icon is to be replaced by @serenamarie125 and her team once they have a font or SVG icon.